src/main/java/edu/wpi/first/shuffleboard/components/TilePane.java

```diff
@@ -29,7 +29,8 @@
 
   private static final int DEFAULT_COL_COUNT = 1;
   private static final int DEFAULT_ROW_COUNT = 1;
-  private static final double MIN_TILE_SIZE = 32; // pixels
+  private static final double MIN_TILE_SIZE = 64; // pixels
+  private static final double MAX_TILE_SIZE = 65535; // pixels
```

Comment:
the default is *negative* infinity, which we should use with Region.USE_PREF_SIZE rather then creating a new constant here.